homeassistant/components/media_player/universal.py

```diff
@@ -328,6 +332,11 @@ def current_source(self):
         return self._child_attr(ATTR_INPUT_SOURCE)
 
     @property
+    def source_list(self):
+        """List of available input sources."""
+        return self._source_list
```

Comment:
That's what I noticed when looking at the code, but a component to create a standalone media player is almost identical to this one, with the exception that it doesn't need children, and every service is implementable with an action.

Do you recommend to create a different component for it then? I just want to specify all actions myself and map them to MQTT messages. A bit like the templated-style components.